lib/package-manager/install.js

```diff
@@ -7,15 +7,17 @@ const createOptions = require('../create-options');
 const spawn = require('../spawn');
 const timeout = require('../timeout');
 
-function install(context, next) {
+function install(packageManager, context, next) {
   const options =
     createOptions(
       path.join(context.path, context.module.name), context);
   let args = ['install', '--loglevel', context.options.npmLevel];
-  context.emit('data', 'info', context.module.name + ' npm:',
-      'npm install started');
+  context.emit('data', 'info', context.module.name + ' ' + packageManager +
```

Comment:
can you change all of these into back tick strings while we are at it?

Reply:
Opened up #611 for it